JitsiConferenceEventManager.js

```diff
@@ -522,27 +523,36 @@ JitsiConferenceEventManager.prototype.setupRTCListeners = function() {
             }
         });
 
-    if (conference.statistics) {
```

Comment:
Is there a case when conference.statistics is not defined?

Reply:
Nope, it is always initialized in JitsiConference. But before that it was initialized after calling this method, so we were failing to register these listeners.